tasks/utils/github.js

```diff
@@ -0,0 +1,70 @@
+const axios = require("axios")
+
+export const createGist = async (githubApiToken, encryptedOffchainSecrets) => {
+  const content = JSON.stringify(encryptedOffchainSecrets)
+
+  const headers = {
+    Authorization: `token ${githubApiToken}`,
+  }
+
+  // construct the API endpoint for creating a Gist
+  const url = "https://api.github.com/gists"
+  const body = {
+    public: true,
+    files: {
+      [`encrypted-functions-request-data-${new Date().now()}.json`]: {
+        content,
+      },
+    },
+  }
+
+  try {
+    const response = await axios.post(url, body, { headers })
+    const gistUrl = response.data.html_url
+    return gistUrl
+  } catch (error) {
+    console.error("Failed to create Gist", error)
```

Comment:
We need to throw again here right? Rather than swallow it
Otherwise output of this function will be undefined